corehq/blobs/migrate.py

```diff
@@ -415,8 +415,12 @@ def get_doc_migrator(self, filename, date_range=None, **kw):
         migrator = super(BackendMigrator, self).get_doc_migrator(filename)
         return _migrator_with_worker_pool(migrator, self.reindexer, **kw)
 
-    def get_document_provider(self):
-        return SqlDocumentProvider(self.iteration_key, self.reindexer)
+    def get_document_provider(self, date_range=None, **kw):
```

Comment:
Is there a reason why this accepts `**kw` but does nothing with those arguments? It is typically preferred to only accept arguments that are expected and raise an error if unexpected arguments are passed.
```suggestion
    def get_document_provider(self, date_range=None):
```